src/lib.rs

```diff
@@ -2657,13 +2657,13 @@ mod tests {
     #[test]
     fn test_data_ptr_alignment() {
         let v = ThinVec::<u16>::new();
-        assert!(v.data_raw() as usize % 2 == 0);
+        assert!(v.data_raw() as usize % core::mem::align_of::<u16>() == 0);
```

Comment:
This is prelude now:
```suggestion
        assert!(v.data_raw() as usize % align_of::<u16>() == 0);
```